compaction/compaction_manager.cc

```diff
@@ -164,6 +164,10 @@ bool compaction_manager::can_register_compaction(replica::table* t, int weight,
     if (!t->get_compaction_strategy().parallel_compaction() && has_table_ongoing_compaction(t)) {
```

Comment:
Hi @bhalevy. perhaps we can remove this commit from this branch and leave it for https://github.com/scylladb/scylla/pull/10990 which is dedicated to fixing the problem described. what's your plan?

Reply:
It is part of https://github.com/bhalevy/scylla/commits/always-register-fully-expired-sstables-for-compaction
I made this PR a draft until the dependencies are resolved.
Currently it has only the last 2 unique commits.

Reply:
thanks for the explanation.